webpack.config.js

```diff
@@ -7,11 +7,15 @@ module.exports = {
   entry: './src/now.js',
   target: 'node',
   externals: [nodeExternals()],
+  devtool: 'source-map',
```

Comment:
@rauchg, just curious (as I was enabling source map support myself for fixing #1010): doesn't this `devtool`setting  mean that source maps are always generated, even in production? Looking at https://webpack.js.org/configuration/devtool/ this setting is ok for production, but in your commit message you wrote something about (small) performance hits and that you only want to enable source maps in development.

Reply:
My approach was around something like:

``` javascript
const isDev = process.env.NODE_ENV === 'development';

module.exports = {
  entry: './src/now.js',
  devtool: isDev ? 'source-map' : false,
// [...]
```

(... or introducing https://github.com/kentcdodds/webpack-config-utils if more prod/dev differences within the webpack config pop up.)